open-company/open-company-web
# Preload main 4 containers

#### Review after #1094 

Re-add sections preload but only for Unread, All Posts, Bookmarks and Drafts.

To test:
- go to Unread
- do you see the post appear w/o any special delay?
- do you see AP, Bookmarks and Drafts being loaded with a small delay (5 seconds plus 1 seconds btw each other)
- go to AP
- do you see the post appear w/o any special delay?
- do you see Unread, Bookmarks and Drafts being loaded with a small delay (5 seconds plus 1 seconds btw each other)
- go to Bookmarks
- do you see the post appear w/o any special delay?
- do you see Unread, All Posts and Drafts being loaded with a small delay (5 seconds plus 1 seconds btw each other)
- go to Drafts
- do you see the post appear w/o any special delay?
- do you see Unread, All Posts and Drafts being loaded with a small delay (5 seconds plus 1 seconds btw each other)
- do you NOT see those sections being preloaded again when you do internal nav?
